test/jdk/javax/swing/plaf/nimbus/8041642/bug8041642.java

```diff
@@ -50,17 +60,24 @@ public void run() {
                 }
             });
             final Robot robot = new Robot();
-            robot.delay(300);
+            robot.waitForIdle();
+            robot.delay(1000);
             SwingUtilities.invokeAndWait(new Runnable() {
                 @Override
                 public void run() {
                     point = bar.getLocationOnScreen();
                 }
             });
             Color color = robot.getPixelColor(point.x + 1, point.y + 7);
-            System.out.println(color);
+            System.out.println("point " + point + " color " + color);
             if (color.getGreen() < 150 || color.getBlue() > 30 ||
                     color.getRed() > 200) {
+                Rectangle bounds = frame.getBounds();
+                BufferedImage img = new BufferedImage(bounds.width, bounds.height, BufferedImage.TYPE_INT_ARGB);
+                Graphics g = img.getGraphics();
+                frame.paint(g);
+                g.dispose();
```

Comment:
On the other hand, it doesn't show what's actually happening on the screen, so this image may be perfect, but the window may actually be obscured by something else on the system.

Reply:
When I ran the full swing tests in those system, this test failed and screen.png shows empty desktop so nothing was there to obscure, it seemed to me frame was made visible programmatically but actually there was some latency to make it visible on screen (since otherwise getLocationOnScreen would have returned ISE), so I added this..

Also, this saved image might also help in finding issue if there is indeed no green color padding